src/Item/Item.model.ts

```diff
@@ -68,6 +68,24 @@ export class Item extends Model<ItemAttributes> {
         WHERE ${where}`)
   }
 
+  static async isURNRepeated(
+    id: string,
+    thirdPartyId: string,
+    urnSuffix: string
+  ): Promise<boolean> {
+    const counts = await this.query<{ count: number }>(SQL`
```

Comment:
Count is actually a string, we need to fix other counts too. It's a string because it's a big number.
We also need to take a look at other queries that use count to be sure they work properly.
```suggestion
    const counts = await this.query<{ count: number }>(SQL`
```

Reply:
You're right, the `count` coming from the query is actually a string. This is just a TS typing issue as they are working properly because JS automatically converts the string to a number when doing the check `counts[0].count >0`
![image](https://user-images.githubusercontent.com/8763687/163964016-af10684c-b562-4b9f-9540-fe6ad1eedfca.png)

I'just changed the type to `string` and converted it to `Number` before the check.